includes/Channel.hpp

```diff
@@ -54,6 +55,7 @@ class Channel {
   Mode _mode;
   std::string _key;
   unsigned int _limit;
+  std::set<std::string> _bannedUsers;
```

Comment:
In my opinion, our Server shouldn't have "a ban list" as explained in my comment without supporting 'MODE +b'.
If we would like to add it, it must include more clients information (not only nickname, but also user name and host name) even if we cannot simulate hostname differences in our environment.